### DISCUSSION TITLE
DELETED

### DISCUSSION THREAD

**Discussion by Mani4D46** [wezterm/wezterm#6169] [General]
DELETED

**Comment by loops** [comment]
Hi Mani4D46,

You can get larger characters, using your font, on a line with an escape sequence:

    printf '\e#6Big characters  ##5 #3 #5  !!5% !!#53#3#335'

Although that doesn't really explain what you're seeing with larger characters superimposed over smaller ones.
I wonder if you're hitting a bug with an extremely long line of characters  (no newline)?

But you are likely to see weird things whenever you "cat" binary files to the terminal.  It's much
better to use a command like "xxd" or "hexdump" to view them in a sane way.

**Reply by j4james** [reply]
@Mani4D46 In case you're still curious to know, these are standard escape sequences originally from the VT100 line of terminals. You can choose between double width lines ([`DECDWL`](https://vt100.net/docs/vt510-rm/DECDWL.html)) or both double width and double height ([`DECDHL`](https://vt100.net/docs/vt510-rm/DECDHL.html)).

Technically WezTerm's implementation is not quite correct, because if you're just doing `\e#3`, you should only see the top half of the double-height characters (so there shouldn't be any overlap with the line below). Similarly, if you were using `\e#4`, you should only see the bottom half. But in most real world scenarios those sequences would be used as a pair, with matching text on both lines, so it wouldn't usually be a problem.
